tests/queries/0_stateless/02668_parse_datetime.sql

```diff
@@ -233,5 +233,11 @@ select parseDateTime64('2300-01-01 00:00:00', '%Y-%m-%d %H:%i:%s'); -- { serverE
 
 -- -------------------------------------------------------------------------------------------------------------------------
 
+-- Test parseDateTime should ignore leading spaces with %e
```

Comment:
No, %e for parseDateTime should not ignore leading spaces, it should accept both single and double digit inputs.

Reply:
Yes, it can already accept single and double digit inputs, but the setting is meant to also enable cases where multiple whitespaces are added as a prefix to the single digit input. This is something mysql does : [link](https://onecompiler.com/mysql/43est6a62)